.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,15 +1,22 @@
 ---
 name: Bug Report
 about: Report a bug/crash
-title: ''
+title: '[Bug] A brief description about problem you found'
 labels: ''
 assignees: ''
 
 ---
 
+<!--
 **IMPORTANT:** Before continuing, please ensure that Building Gadgets (and any other mods involved) is updated to the latest available version
 ----------------------------------------------------------------------------
 After deleting this template, please provide the following information:
 * Forge Version
-* *If updating was not possible*; Building Gadgets version
+* Building Gadgets version 
+  * **NEVER** put "lastest", new releases might come out as you post the issue
```

Comment:
```suggestion

```
This text should be kept to a minimum. The more the text, the less likely people are to read it (and to do so thoroughly). There has never been a problem with people typing 'latest', and it's not worth an extra line for the rare chance someone would in the future.